traitsui/tests/editors/test_check_list_editor.py

```diff
@@ -0,0 +1,512 @@
+import unittest
+
+from pyface.gui import GUI
+
+from traits.api import HasTraits, List, Str
+from traitsui.api import CheckListEditor, UItem, View
+from traitsui.tests._tools import (
+    is_current_backend_qt4,
+    is_current_backend_wx,
+    skip_if_null,
+    store_exceptions_on_all_threads,
+)
+
+
+class ListModel(HasTraits):
+
+    value = List()
+
+
+def get_view(style):
+    return View(
+        UItem(
+            "value",
+            editor=CheckListEditor(
+                values=["one", "two", "three", "four"],
+            ),
+            style=style,
+        ),
+        resizable=True
+    )
+
+
+def get_mapped_view(style):
+    return View(
+        UItem(
+            "value",
+            editor=CheckListEditor(
+                values=[(1, "one"), (2, "two"), (3, "three"), (4, "four")],
+            ),
+            style=style,
+        ),
+        resizable=True
+    )
+
+
+def get_combobox_text(combobox):
+    """ Return the text given a combobox control """
+    if is_current_backend_wx():
+        return combobox.GetString(combobox.GetSelection())
+
+    elif is_current_backend_qt4():
+        return combobox.currentText()
+
+
+def set_combobox_index(editor, idx):
```

Comment:
For wx the function is essentially the same as in `test_enum_editor` so I would like to keep the name the same. The difference is that I can't test setting the index for qt and have to pass in the editor rather than combobox control to do an equivalent test. I update the docstring, hopefully it explains it better.

Reply:
Yep. It helps. Thank you!